src/transaction/process.rs

```diff
@@ -2,6 +2,7 @@ use std::sync::{mpsc::Receiver, Arc, RwLock};
 
 use crate::blockcfg::{BlockConfig, Ledger, Transaction};
 use crate::blockchain::BlockchainR;
+use crate::client::do_stream_reply;
```

Comment:
I think `do_stream_reply` should be moved to `intercom`, now that many tasks use this helper function.